src/docs/gold_standard_slot_examples.md

```diff
@@ -0,0 +1,444 @@
+# Gold Standard MIxS LinkML Slots Examples
+
+## Introduction
+
+This page lists a range of 'gold standard' examples of slots that are used in the MIxS LinkML schema.
+
+There are examples of slots that cover a range of common types of data, including:
+
+- Basic slots (i.e., match exactly to a LinkML `range` type):
+  - [Integer](#integer) (numeric)
+  - [Float](#float) (numeric)
+  - [String](#string) (free text)
+  - [Fixed list](#fixed-list) (enumeration)
+  - [Boolean](#boolean)
+- Complex (i.e., common more complex `range` types that are more specific to MIxS):
+  - [Structured text (text only)](#structured-text-text-only) (text only)
+  - [Structured text (with unit, measurement)](#structured-text-numeric-with-unit-measurement) (numeric with units)
+  - [Structured text (ontology term)](#structured-text-ontology-term)
+  - [URL](#url) (or other link utilising a URL like structure)
+  - [Mixed type (free text or ontology)](#mixed-type) (either free text, or from an ontology)
+
+You can use these as templates on how to construct your own new slot to the highest quality level expected.
+
+## Overview
+
+Every MIxS LinkML slot should have at a minimum the following attributes:
+
+- [`description`](https://linkml.io/linkml/schemas/metadata.html#providing-descriptions): the description of what the metadata term is for
+- [`title`](https://linkml.io/linkml-model/latest/docs/title/): a short human readable 'title' for the slot
+- [`examples`](https://linkml.io/linkml-model/latest/docs/examples/): examples values demonstrating how the slot should be used
+- [`in_subset`](https://linkml.io/linkml-model/latest/docs/in_subset/): the section of the schema that the slot belongs based on a [fix list of MIxS categories](https://github.com/GenomicsStandardsConsortium/mixs/blob/609b0f567486f64cb7061246588d8006f87fa138/src/mixs/schema/mixs.yaml#L21-L26)
```

Comment:
I'm not sure we would want a contributor to determine the subset (categories). I also believe @only1chunts is working on some changes to these. So may be best to take this out, and make a issue to add this when it's cleared up.

Reply:
also, just saw the note! :) 
So we could keep it

Reply:
Yes, I'm basing the current document on what we _currently_ have in the schema, this will of course evolve :)